src/tool/subcommands/api_cmd/api_compare_tests.rs

```diff
@@ -1231,7 +1231,7 @@ fn eth_tests() -> Vec<RpcTest> {
                 None,
                 Some(
                     EthBytes::from_str(
-                        concat!("0x", include_str!("./contracts/invoke_cthulhu.hex")).trim(),
+                        concat!("0x", include_str!("./contracts/cthullu/cthulhu.hex")).trim(),
```

Comment:
some typo in `cthulhu` dir

Reply:
would made more sense to call it `cthulhu/invoke.hex`

Reply:
Done.